components/rss/actions/merge-rss-feeds/merge-rss-feeds.js

```diff
@@ -0,0 +1,94 @@
+const Parser = require('rss-parser');
+
+module.exports = {
+  name: "Merge RSS Feeds",
+  description: "Retrieve multiple RSS feeds and return a merged array of items sorted by date.",
+  key: "rss-merge-rss-feeds",
+  version: "0.2.1",
+  type: "action",
+  props: {
+	feeds: {
+		type:"string[]",
+		label:"Feeds",
+		description:"The list of RSS feeds you want to parse."
+	},
+	merge: {
+		type:"boolean",
+		optional:true,
+		default:true,
+		description:"If true, all items are returned in a date sorted array. If false, each feed is returned as one result in the array."
```

Comment:
You should be able to use Markdown in these descriptions, so I'd recommend surrounding `true` and `false` with single backticks to highlight the fact that they're values.

Reply:
FYI, as I work I work I'm going to resolve, but won't be pushing for each, so if something happens and I get delayed, you may see stuff resolved that isn't pushed to GH.